### DISCUSSION TITLE
Candidatura di: Manuel Ricci

### DISCUSSION THREAD

**Discussion by guizzo** [Il-Libro-Open-Source/book#126] [Candidature Ambassadors In Corso]
Questo sondaggio viene aperto per tutto il mese di Settembre per indicare la preferenza o l'opposizione sulla candidatura a ruolo di Ambassador per:

[Manuel Ricci](https://github.com/manuelricci)

In caso di disaccordo, si protegge l'anonimato ma sarebbe gradita una spiegazione tramite commento a questo sondaggio con delle motivazioni.

Attualmente, vista la fase embrionale del progetto, non ci saranno limiti minimi di votazione e la semplice superiorità dei voti a favore sarà considerata sufficiente a procedere con la nomina.

**Comment by chiara1982** [comment]
Ho avuto Manuel come insegnante di JavaScript all’interno del corso di Web Design organizzato da DotAcademy nel 2019. Molto preparato ed anche una brava persona. Mi ha aiutata anche dopo il corso a risolvere un problema di codici. Ha tutto il mio appoggio, bravo Manuel! 💚

**Comment by eppak** [comment]
Ciao @manuelricci, 
Ti confermiamo che la votazione si chiude oggi e che è positiva. Benvenuto tra gli Ambassador, ti chiediamo di aggiungere Michael su Telegram «at»Cadienvan che provvederà ad aggiungerti al gruppo organizzativo. 
Ti ricordiamo di prendere visione, oltre delle linee guida, anche del nostro codice di condotta che trovi https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md. 
Il repository sulla governance invece lo trovi a questo indirizzo dove sono presenti anche i gruppi di lavoro. 
Ti aspettiamo!
